### PR TITLE
Update database.js

### PR DESCRIPTION
**Pull request by vitaly-t**
better use of `pg-promise`. also, looks like in many places you are mistakenly use `db.one` instead of `db.oneOrNone` :wink:

### CODE REVIEW

File: server/database.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ exports.deleteDeck = function(db, deck_id) {
 }
 
 exports.updateDeck = function(db, deck) {
-  return db.tx(t => {
+  return db.tx('update-deck', t => {
```
Comment (vitaly-t) [inline server/database.js]: task should be used for read-only blocks ;)

File: server/database.js [hunk 2]
```diff
@@ -20,23 +20,23 @@ exports.connectToDatabase = function() {
...
+  return db.any(`
     SELECT d.id, d.title, u.name AS author FROM 
     decks d INNER JOIN users u ON (d.author = u.id) WHERE d.author = $1
     ORDER BY d.title ASC
   `, user_id)
 }
 
 exports.getDeck = function(db, deck_id) {
-  return db.tx(t => {
+  return db.task('get-deck', t => {
```
Comment (vitaly-t) [inline server/database.js:31]: task should be used for read-only blocks.

**Review by sequoian — APPROVED** [review]
Thank you for the feedback!  I'm curious why you added labels to the tasks and transactions; I couldn't find anything in your documentation about doing that.

**Comment by vitaly-t** [comment]
see wiki:

* https://github.com/vitaly-t/pg-promise/wiki/Tags
* https://github.com/vitaly-t/pg-promise/wiki/chaining-queries

**Comment by sequoian** [comment]
Thanks again!

**Pull request merged**